game.libretro.scummvm/addon.xml

```diff
@@ -13,8 +10,11 @@
                library_windx="game.libretro.scummvm.dll"
                library_wingl="game.libretro.scummvm.dll">
         <platforms></platforms>
-        <extensions></extensions>
+        <extensions>scummvm|scum</extensions>
         <supports_vfs>false</supports_vfs>
+        <supports_game_loop>true</supports_game_loop>
+        <supports_standalone>true</supports_standalone>
```

Comment:
Yes, that makes sense. There might be other areas where the the code assumes that standalone is exclusive with launching games by extension, the add-on manager comes to mind